values.yaml

```diff
@@ -222,7 +222,8 @@ minio:
     size: 10Gi
 
   resources:
-    requests: null ## https://github.com/helm/helm/issues/1966
+    requests: ## https://github.com/helm/helm/issues/1966
+      memory: 1Gi # By default minio request 4Gi of memory. Set it to 4 only if you have enough memory on your cluster
```

Comment:
@nandiheath how come `request: null` does not invalidate the requests in minio?

Reply:
I don't know... When I check the `deploy`/`pod` after a fresh install it has `requests: 4Gi`. And I found it is the default value of the helm chart for minio

Reply:
Yeah but with `null` it should not do it.

What helm version are you using? Gonna try?

Reply:
`null` works on helm `3.1.3` and `3.1.2`, but not working on `3.3.0`
so I suspect it is helm related issue

Reply:
Ok will need to test it then, that's not good.